vlasovsolver/cpu_trans_map_amr.cpp

```diff
@@ -1953,36 +672,36 @@ void update_remote_mapping_contribution_amr(
                  3) Ref_nbr >  Ref_c     , index = nbr sibling index
                  4) Ref_nbr <  Ref_c     , index = c   sibling index
                 */
-                                             
+
                if(mpiGrid.get_refinement_level(nbr) >= mpiGrid.get_refinement_level(c)) {
 
                   // Allocate memory for one sibling at recvIndex.
-                  
+
                   recvIndex = get_sibling_index(mpiGrid,nbr);
 
                   ncell->neighbor_number_of_blocks.at(recvIndex) = ccell->get_number_of_velocity_blocks(popID);
                   ncell->neighbor_block_data.at(recvIndex) =
-                     (Realf*) aligned_malloc(ncell->neighbor_number_of_blocks.at(recvIndex) * WID3 * sizeof(Realf), 64);
+                     (Realf*) aligned_malloc(ncell->neighbor_number_of_blocks.at(recvIndex) * WID3 * sizeof(Realf), WID3);
```

Comment:
If someone gets odd ideas, will this crash and burn?

Reply:
No, it just aligns the data based on the block size, which is now correct. We still want alignment even if we change WID3.